### PR TITLE
Rewrite most of the FAQ

### PR DESCRIPTION
**Pull request by jsoref**

### CODE REVIEW

**Comment by jsoref** [comment]
If someone approves the changes, I'll force push to fix the DCO. I'm disappointed that github.dev didn't DCO it for me, I thought they fixed that.

File: versioned_docs/version-latest/faq.md [hunk 1]
```diff
@@ -6,14 +6,14 @@ This FAQ is a work in progress designed to answer the questions our users most f
 
 #### **Q: Is Rancher Desktop a desktop version of Rancher?**
 
-**A:** While [Rancher](https://rancher.com/) and [Rancher Desktop](https://rancherdesktop.io/) share the _Rancher_ name, they do different things. Rancher Desktop is not Rancher on the Desktop. Rancher is a powerful solution to manage Kubernetes clusters. Rancher Desktop runs local Kubernetes and a container management platform. The two solutions complement each other.
+**A:** While [Rancher](https://rancher.com/) and [Rancher Desktop](https://rancherdesktop.io/) share the _Rancher_ name, they do different things. Rancher Desktop is not Rancher on the Desktop. Rancher is a powerful solution to manage Kubernetes clusters. Rancher Desktop allows users to locally run a container management platform optionally including Kubernetes. The two solutions complement each other.
```
Comment (gunamata) [inline versioned_docs/version-latest/faq.md:9]:
```suggestion
**A:** While [Rancher](https://rancher.com/) and [Rancher Desktop](https://rancherdesktop.io/) share the _Rancher_ name, they do different things. Rancher Desktop is not Rancher on the Desktop. Rancher is a complete software stack for managing multiple Kubernetes clusters across any infrastructure. On the other hand, Rancher Desktop provides Kubernetes and container management on the desktop to support local development use cases. The two solutions complement each other.
```

File: versioned_docs/version-latest/faq.md [hunk 2]
```diff
@@ -97,20 +97,26 @@ kubectl apply -f expose-traefik.yaml
...
+#### **Q: Can I disable Traefik, and will doing so remove Traefik resources?**
 
 **A:** Yes, you can disable Traefik in order to free up port 80 and 443 for alternate ingress configuration. Disabling Traefik will _not_ delete existing resources. By default, the `Enable Traefik` feature is selected under **Kubernetes Settings**; uncheck this box to disable it.
 
 If you want to delete Traefik resources, click on `Reset Kubernetes` on the **Kubernetes Settings** panel of the UI.
 
 #### **Q: Is there support for internal container port forwarding?**
 
-**A:** Yes, support has been enabled again for this feature on Linux and macOS.
+**A:** Yes, support for this feature on Linux and macOS is available as of [v1.1.0](https://github.com/rancher-sandbox/rancher-desktop/releases/tag/v1.1.0).
```
Comment (gunamata) [inline versioned_docs/version-latest/faq.md:108]:
```suggestion
**A:** Yes, support for this feature is available on all three platforms, Windows, Linux, and macOS, as of [v1.1.0](https://github.com/rancher-sandbox/rancher-desktop/releases/tag/v1.1.0).
```

**Review by gunamata — CHANGES_REQUESTED** [review]
@jsoref , Thank you for putting together this PR. The changes look good to me except for 2 minor phrasing suggestions.

File: versioned_docs/version-latest/faq.md (hunk 1 above)
Comment (jsoref) [reply versioned_docs/version-latest/faq.md:9]: Hmm. The first half is definitely fine.

The second part bothers me. I know that some people use Rancher Desktop for Kubernetes, but most of my team just uses it for Docker, i.e. we've turned off Kubernetes. (The fact that my team is slowly turning it back on is out of scope for this thought.)

I'm also going to reword the thing to start with "No."

```suggestion
**A:** No. Rancher Desktop is not Rancher on the Desktop. 

While [Rancher](https://rancher.com/) and [Rancher Desktop](https://rancherdesktop.io/) share the _Rancher_ name, they do different things. But, the two solutions do complement each other:

Rancher is a complete software stack for managing multiple Kubernetes clusters across any infrastructure.

Rancher Desktop provides container management and an optional Kubernetes instance on the desktop to support local development use cases.
```

**Comment by jsoref** [comment]
The DCO hates me, so if @gunamata is ok w/ these changes, I'll rebase w/ proper DCOs to make it happy. If not, we can iterate on the text before I spend time dealing w/ it.

File: versioned_docs/version-latest/faq.md [hunk 3]
```diff
@@ -6,7 +6,13 @@ This FAQ is a work in progress designed to answer the questions our users most f
...
 #### **Q: Is Rancher Desktop a desktop version of Rancher?**
 
-**A:** While [Rancher](https://rancher.com/) and [Rancher Desktop](https://rancherdesktop.io/) share the _Rancher_ name, they do different things. Rancher Desktop is not Rancher on the Desktop. Rancher is a powerful solution to manage Kubernetes clusters. Rancher Desktop allows users to locally run a container management platform optionally including Kubernetes. The two solutions complement each other.
+**A:** No. Rancher Desktop is not Rancher on the Desktop. 
+
+While [Rancher](https://rancher.com/) and [Rancher Desktop](https://rancherdesktop.io/) share the _Rancher_ name, they do different things. But, the two solutions do complement each other:
+
+Rancher is a complete software stack for managing multiple Kubernetes clusters across any infrastructure.
+
+Rancher Desktop provides container management and an optional Kubernetes instance on the desktop to support local development use cases.
```
Comment (gunamata) [inline versioned_docs/version-latest/faq.md:15]: The use of `optional`, even though technically correct, deemphasizes the fact that Rancher Desktop takes a Kubernetes first approach, and sounds like Kubernetes is available as just an option. Can you please remove the word `optional`?

Comment (jsoref) [reply versioned_docs/version-latest/faq.md:15]: sold

**Comment by jsoref** [comment]
@gunamata: reorganized/folded/cleaned up messages, even `git` likes the branch more now, and the DCO is happy.

**Review by gunamata — APPROVED** [review]
LGTM

**Pull request merged**